leapdao/nervos
# Add relay service listeners and handlers

### How to test

1. Start ethermint
2. Start Redis on port 6379
3. In the relayer folder add childchain bridge address.
4. If not deployed you can run `ts-node deploy.ts`. Copy value of address from address.json to key address in config.json
5. Run `ts-node start.ts`. 
6. Test Unlock by running `ts-node /src/utils/transaction.ts`. Will create 3 separate burn events that should be captured and recorded

Closes #26 Closes #40
